### ISSUE TITLE
[QUERY] Azure Table - Authenticate with AAD

### ISSUE THREAD

**Issue by abhikt48** [Azure/azure-sdk-for-java#16388]
New SDK - Do we have any option to authenticate Table Client with AAD(Azure Active Directory)? 

If yes, could you please provide any sample?

**Comment by bsiegel** [comment]
Hey again @abhikt48 - we will definitely support authenticating with AAD as a critical scenario. The basic infrastructure is there for this to work today, but it looks like some of the specifics needed for AAD still need to be completed. I've filed #16397 to track this and make sure it works .

**Comment by bsiegel** [comment]
It looks like there's no further action required on this issue so I'm going to close it for now. Please track the above linked issue to see how progress is coming on AAD authentication, and if you have questions in the future please don't hesitate to open a new issue!

**Comment by bsiegel** [comment]
Just a quick update here, the library has support for accepting an AAD credential. However, the underlying Azure Tables service does not yet support authenticating with AAD. The service team does plan to support AAD in the future but there is not currently a timeframe for that. So I'll make sure that once the feature is released our library is ready to authenticate using an AAD credential, but it won't actually be possible to use such a credential until the service itself supports AAD auth.
